### DISCUSSION TITLE
DevTools not working after upgrade to v6?

### DISCUSSION THREAD

**Discussion by mortmoe** [react-hook-form/react-hook-form#2021] [Q&A] [answered]
Isn't devtools compatible with v6? No changes in code (except I updated DevTools to 1.2.1) and I'm getting these error messages: 

Uncaught TypeError: formState.dirtyFields.has is not a function
    at devtools.js:288
    at Array.map (<anonymous>)
    at Panel (devtools.js:281)
    at renderWithHooks (react-dom.development.js:14825)
    at updateFunctionComponent (react-dom.development.js:17058)
    at beginWork (react-dom.development.js:18643)
    at HTMLUnknownElement.callCallback (react-dom.development.js:188)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:237)
    at invokeGuardedCallback (react-dom.development.js:292)
    at beginWork$1 (react-dom.development.js:23234)

**Comment by bluebill1049** [comment, marked as answer]
@mortmoe i will release a new major version. forgot devTool.

**Reply by mortmoe** [reply]
Oh, ok, not just me then :D Thanks for the quick response. Reason I wanted it actually another problem with the reset. My form now clears the values when I call reset instead of setting the defaultValues. Is there any changes I need to do related to the reset function?

**Reply by bluebill1049** [reply]
do you have a codesandbox?

**Reply by bluebill1049** [reply]
The devtool has been fixed `2.0.1`

**Reply by bluebill1049** [reply]
thanks for the notice: https://codesandbox.io/s/react-hook-form-template-ttzn8?file=/src/App.tsx

**Comment by mortmoe** [comment]
Cool, I'll check it out. Made a quick codesandbox for the issue with reset. It seems like it works normally when I put the mui component inside controllers. Also setting defaultValue seems to have something to say, but notice in the sandbox what happens if I set the TextField defaultValue to {getValue('test3)} and then do reset. Everything seems ok but... if you submit the value is totally gone. Also I cannot figure out what happens in test4 if you remove ... seems to remove everything (as if I had forgotten to do register() instead of register).

https://codesandbox.io/s/react-hook-form-reset-problem-9qzzj?file=/src/index.js

**Comment by mortmoe** [comment]
@bluebill1049 I did some more testing on this and created a new post: https://github.com/react-hook-form/react-hook-form/discussions/2043
